src/main/java/io/cloudtrust/keycloak/representations/idm/UsersPageRepresentation.java

```diff
@@ -0,0 +1,45 @@
+package io.cloudtrust.keycloak.representations.idm;
+
+import org.keycloak.representations.idm.UserRepresentation;
+
+import java.util.List;
+
+public class UsersPageRepresentation {
+
+    /**
+     * Users
+     */
+    List<UserRepresentation> users;
+
+    /**
+     * Total users count for those search criteria
```

Comment:
There is no mention of any search criteria here... I would simplify as `Total users count`

Reply:
Fixed